plugin-template

```diff
@@ -217,6 +218,28 @@ def main():
                         """
         ),
     )
+    parser.add_argument(
+        "--add-version-ci-update-branches",
```

Comment:
Not sure, the other one is actually more descriptive...
```suggestion
        "--add-ci-branch",
```

So am i understanding correctly that this is another option like `--github` or `--docs` that specifies what parts of the plugin to change? I guess, it implies `--github`?

Reply:
The option only modifies the variable in the config file, you have to use `--github` in order to update the templated files.

Reply:
So yes in a way that it implies you also want to specify `--github`. XD